### PR TITLE
update readme for TO on cross compilation

### PR DESCRIPTION
**Pull request by asheikm**
<hr>

## Which Traffic Control components are affected by this PR?

- Documentation

## What is the best way to verify this PR?
This PR updates README doc, 

## PR submission checklist
- [x] This PR has documentation

### CODE REVIEW

**Comment by asheikm** [comment]
I tried to cross compile on windows for CentOS x64, hope this instruction would help it ? @zrhoffman pls review

File: traffic_ops/traffic_ops_golang/README.md [hunk 1]
```diff
@@ -79,6 +79,14 @@ Go is a compiled language so any local changes will require you to CTRL-C the co
 
 `go build && ./traffic_ops_golang -cfg $HOME/cdn.conf -dbcfg ../app/conf/development/database.conf`
 
+To cross compile the code in windows to generate binaryt to run on linux, Please replace GOARCH for architecture to build
```
Comment (ocket8888) [inline traffic_ops/traffic_ops_golang/README.md:82]: spelling; "binaryt" -> "binary"

Windows and Linux are proper nouns; need capitalization

Grammar: "Please" should not be capitalized in the middle of the sentence

Grammar and clarity: "..., Please replace GOARCH for architecture to build" -> "..., please run the following command, replacing `GOARCH` with the architecture for which to build." (note ending punctuation).

File: traffic_ops/traffic_ops_golang/README.md [hunk 2]
```diff
@@ -79,6 +79,14 @@ Go is a compiled language so any local changes will require you to CTRL-C the co
...
+
+`env GOOS=linux GOARCH=arm64 go build `
```
Comment (ocket8888) [inline traffic_ops/traffic_ops_golang/README.md:84]: This should probably be in a code block like:
~~~markdown
```bash
env GOOS=linux GOARCH=arm64 go build
```
~~~

File: traffic_ops/traffic_ops_golang/README.md [hunk 3]
```diff
@@ -79,6 +79,14 @@ Go is a compiled language so any local changes will require you to CTRL-C the co
...
+
+Once binary generated, can be copied into linux machine to run
```
Comment (ocket8888) [inline traffic_ops/traffic_ops_golang/README.md:86]: Linux is a proper noun, needs capitalization.

Grammar: "Once <ins>the</ins> binary <ins>is</ins> generated, <ins>it</ins> can be copied <del>into</del><ins>onto</ins> <ins>a</ins> <del>l</del><ins>L</ins>inux machine to <ins>be</ins> run<ins>.</ins>" (note ending punctuation).

**Review by ocket8888 — CHANGES_REQUESTED** [review]

**Review by ocket8888 — APPROVED** [review]

**Pull request merged**